src/Config/RuntimeConfigLoader.cs

```diff
@@ -96,6 +105,57 @@ public static bool TryParseConfig(string json, [NotNullWhen(true)] out RuntimeCo
         return true;
     }
 
+    /// <summary>
+    /// Parses a JSON string into a <c>RuntimeConfig</c> object for multiple database scenario.
+    /// </summary>
+    /// <param name="json">JSON that represents the config file.Json expected to contain Datasources dictionary.</param>
+    /// <param name="config">The parsed config, or null if it parsed unsuccessfully.</param>
+    /// <returns>True if the config was parsed, otherwise false.</returns>
+    public static bool TryParseConfigMultipleDatabase(string json, [NotNullWhen(true)] out RuntimeConfig? config, ILogger? logger = null, Dictionary<string, string>? datasourceNameToConnectionString = null)
```

Comment:
This function replicates the functionality of the `TryParseConfig` function except that it missed the adding application name for telemetry for MsSQL. Was that intentional?

Reply:
good catch. fixed.